Tasks/AzureStorage/main.ts

```diff
@@ -0,0 +1,45 @@
+import path = require("path");
+import * as os from 'os';
+import util = require("util");
+import tl = require("vsts-task-lib/task");
+import msRestAzure = require("azure-arm-rest/azure-arm-common");
+import armStorage = require("azure-arm-rest/azure-arm-storage");
+import blobUtils = require("./operations/AzureBlobUtils");
+
+function run(): Q.Promise<void> {
+    var artifactsDirectory = tl.getVariable('System.ArtifactsDirectory');
```

Comment:
will azcopy clean folder before downloading? This could be a problem as there could be other artifacts as well. should we have a sub-folder with artifact alias? I think artifacts are downloaded under alias sub-folder.